### ISSUE TITLE
cannot read property name of null (fpSpecified) on v2.2.0

### ISSUE THREAD

**Issue by nfcampos** [lodash/babel-plugin-lodash#38]
Hi,

I'm getting this error on v2.2.0 (but not on 2.1.0)
```
ERROR in ./frontend/js/entry.js
Module build failed: TypeError: /Users/nuno/cp/james/frontend/js/entry.js: Cannot read property 'name' of null
    at PluginPass.<anonymous> (/Users/nuno/cp/james/node_modules/babel-plugin-lodash/lib/index.js:152:28)
    at newFn (/Users/nuno/cp/james/node_modules/babel-core/node_modules/babel-traverse/lib/visitors.js:262:19)
    at NodePath._call (/Users/nuno/cp/james/node_modules/babel-core/node_modules/babel-traverse/lib/path/context.js:63:18)
    at NodePath.call (/Users/nuno/cp/james/node_modules/babel-core/node_modules/babel-traverse/lib/path/context.js:47:17)
    at NodePath.visit (/Users/nuno/cp/james/node_modules/babel-core/node_modules/babel-traverse/lib/path/context.js:93:12)
    at TraversalContext.visitQueue (/Users/nuno/cp/james/node_modules/babel-core/node_modules/babel-traverse/lib/context.js:146:16)
    at TraversalContext.visitMultiple (/Users/nuno/cp/james/node_modules/babel-core/node_modules/babel-traverse/lib/context.js:110:17)
    at TraversalContext.visit (/Users/nuno/cp/james/node_modules/babel-core/node_modules/babel-traverse/lib/context.js:176:19)
    at Function.traverse.node (/Users/nuno/cp/james/node_modules/babel-core/node_modules/babel-traverse/lib/index.js:131:17)
    at NodePath.visit (/Users/nuno/cp/james/node_modules/babel-core/node_modules/babel-traverse/lib/path/context.js:103:22)
```
I can reproduce this error with a file as simple as this (no use of lodash in the file)
```js
for (const a of b) {}
```

Thanks,
Nuno

**Comment by ablipan** [comment]
Meet the problem either after upgrade to v2.2.0.

**Comment by Bertg** [comment]
+1

**Comment by gabro** [comment]
Same here. Could it be that the issue is exactly the lack of a lodash import in the file?

**Comment by danez** [comment]
Same here with imports like that: `import { without } from 'lodash';`

**Comment by megawac** [comment]
Fixed in 2.2.1

**Comment by danez** [comment]
thanks @megawac
